gads-ui/src/services/axios.js

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios'
+
+export function getAPIClient(ctx) {
+    const api = axios.create({
+        baseURL: `http://${process.env.REACT_APP_PROVIDER_HOST}`
```

Comment:
Do we actually need this, when we are serving the static files through the Go app then we shouldn't have to set this. Won't just:
```suggestion
        baseURL: ``
```
work? :D